new/PRS14159TanseaKrestos.xml

```diff
@@ -0,0 +1,62 @@
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14159TanseaKrestos" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ተንሥአ፡ ክርስቶስ፡</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-10-19">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ተንሥአ፡ ክርስቶስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Tanśəʾa Krəstos</persName>
+                    <persName><roleName type="title">liqa ʾabaw</roleName></persName>
+                    <floruit notBefore="1650" notAfter="1750">Second half of 17th or first half of 18th century</floruit>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS14159TanseaKrestos" passive="Tanasee1#a30"/>
+                    <relation name="dc:relation" active="PRS14159TanseaKrestos" passive="PRS14081TanseaKrestos"><desc>Possibly 
+                        the same as <persName ref="PRS14081TanseaKrestos"/>, who is mentioned as a nəbura ʾəd in the same codex in 
+                        <ref target="Tanasee1#12"></ref></desc></relation>
```

Comment:
reference to entitites external to this recod should be made as `<ref type="mss" corresp="Tanasee1"/>`

Reply:
I changed to the phrase, that you suggested and added the folio numbers. Without pointing to the exact addition (a12), the sentence would be otherwise incomprehensive.